solutions/page_object/pages/base_page.py

```diff
@@ -0,0 +1,45 @@
+import math
+import time
+
+import pytest
+from selenium.common.exceptions import (
+    NoSuchElementException,
+    TimeoutException,
+)
```

Comment:
Кстати, тут проще теперь можно, если хочешь.

```suggestion
from selenium.common.exceptions import NoSuchElementException, TimeoutException
```